src/Hng.Infrastructure/Context/MyDBContext.cs

```diff
@@ -12,5 +13,26 @@ public class MyDBContext : DbContext
         public MyDBContext(DbContextOptions<MyDBContext> options) : base(options)
         {
         }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+
+             modelBuilder.Entity<User>()
+                .HasMany(ur => ur.OrganisationUsers)
+                .WithOne(u => u.User)
+                .HasForeignKey(f => f.UserId)
+                .IsRequired();
+
+                 modelBuilder.Entity<User>()
```

Comment:
@davideshett This is repeated and it's not actually necessary. Please review how to create a many to many relationship with EF core and correct this.